src/main/java/org/hydev/mcpm/client/arguments/CommandsFactory.java

```diff
@@ -44,9 +46,16 @@ public static List<CommandParser> baseParsers() {
         var searchController = new SearchPackagesController(searcher);
         var mirrorController = new MirrorController(new MirrorSelector());
         var infoController = new InfoController(tracker);
+        PluginLoader pluginLoader = null;
+        if (isMinecraft) {
+            pluginLoader = new PluginLoader();
+        }
+        DatabaseManager databaseManager = new DatabaseManager(tracker, searcher);
+        System.out.println(isMinecraft);
         var installController = new InstallController(new InstallInteractor(
             new SpigotPluginDownloader(new Downloader(), host.toString()),
-            new DatabaseManager(tracker, searcher)));
+            databaseManager, pluginLoader));
```

Comment:
Can SpigotPluginDownloader handle a null pluginLoader or do we only care if it works in minecraft

Reply:
it should handle a null plugin loader 🤔

Reply:
Or maybe we can create a plugin loader implementation that does nothing?